addons/account_edi/models/account_edi_format.py

```diff
@@ -444,7 +444,7 @@ def _decode_attachment(self, attachment):
         is_text_plain_xml = 'text/plain' in attachment.mimetype and content.startswith(b'<?xml')
         if 'pdf' in attachment.mimetype:
             to_process.extend(self._decode_pdf(attachment.name, content))
-        elif 'xml' in attachment.mimetype or is_text_plain_xml:
+        elif re.match(r'(?:\+)?xml(?![a-zA-Z])', attachment.mimetype) or is_text_plain_xml:
```

Comment:
Maybe remove first part? 
```suggestion
        elif re.match(r'xml(?![a-zA-Z])', attachment.mimetype) or is_text_plain_xml:
```

Reply:
The Purpose of checking '+' ahead was to pass `.xhtml`, `.xul` which have `+xml` in their mime-type and sometimes `.xml` files also can have `application/atom+xml` mime-type. [Source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)

But the regex is not working properly right now because if messed up a little bit. So, do we want to match `+xml` now? If yes then i will update the regex or else i would remove the `+` checking part.

Reply:
You probably want to use `re.search()` instead of `match()` here https://docs.python.org/3/library/re.html#search-vs-match
I'm unsure why do we need to capture the `+`? We want to ensure the string ends with `xml`, right? Or am I missing something? `.xhtml` and `.xul` mimetypes also seem to end with xml.
In that case, `xml$` should work or just `endswith`?

Reply:
The purpose of capturing `+` was to ensure that there is no other character ahead of `xml` to make the regex more secure so that we might not end up capturing any other mime-types which are not XML if such appears in the future.

But I do agree with just checking it to end with `xml` will be enough as per the mime-types containing XML there are currently.

Reply:
Changes Done!